src/components/SelectionList/BaseListItem.js

```diff
@@ -83,7 +83,13 @@ function BaseListItem({
                     )}
                     <ListItem
                         item={item}
-                        isFocused={isFocused}
+                        textStyles={[
+                            styles.optionDisplayName,
+                            isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText,
+                            isUserItem || item.isSelected ? styles.sidebarLinkTextBold : null,
+                            styles.pre,
+                        ]}
+                        alternateTextStyles={[styles.optionAlternateText, styles.textLabelSupporting, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, styles.pre]}
```

Comment:
The order of styles matter. We used to have `styles.textLabelSupporting` passed last so a font size of 13px is applied. But now with the new order another font size is being applied. This caused a regression (partly) https://github.com/Expensify/App/issues/32614

Reply:
Oh, didn't notice it, should I create a PR to fix it?

Reply:
Yes please

Reply:
Okay

Reply:
This is fixed already